Find a functionally equivalent code:android.widget.DatePicker.setMinDate:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the minimal date supported by this {@link NumberPicker} in
 * milliseconds since January 1, 1970 00:00:00 in
 * {@link TimeZone#getDefault()} time zone.
 *
 * @param minDate The minimal supported date.
 */

@@ -1,13 +1,3 @@
 {
-    mTempDate.setTimeInMillis(minDate);
-    if (mTempDate.get(Calendar.YEAR) == mMinDate.get(Calendar.YEAR) && mTempDate.get(Calendar.DAY_OF_YEAR) != mMinDate.get(Calendar.DAY_OF_YEAR)) {
-        return;
-    }
-    mMinDate.setTimeInMillis(minDate);
-    mCalendarView.setMinDate(minDate);
-    if (mCurrentDate.before(mMinDate)) {
-        mCurrentDate.setTimeInMillis(mMinDate.getTimeInMillis());
-        updateCalendarView();
-    }
-    updateSpinners();
+    mDelegate.setMinDate(minDate);
 }

